plugins/InfoBipSmsBundle/Api/SmsInfoBipApi.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2016 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+namespace MauticPlugin\InfoBipSmsBundle\Api;
+
+
+use Mautic\CoreBundle\Factory\MauticFactory;
+use Mautic\CoreBundle\Helper\PhoneNumberHelper;
+use Mautic\PageBundle\Model\TrackableModel;
+
+class SmsInfoBipApi extends AbstractSmsApi
+{
+	private $username;
+	private $password;

```

Comment:
Here and at other places of this file are wrong indentations. A tab instead of four spaces probably. Use the PHP CS Fixer pre-commit hook to format the code for you automatically: https://github.com/mautic/mautic/blob/staging/.github/CONTRIBUTING.md#coding-standards